src/seedu/addressbook/commands/Command.java

```diff
@@ -26,6 +26,13 @@ public Command(int targetIndex) {
     protected Command() {
     }
 
+    /**
```

Comment:
A more precise phrasing is `Returns true if …` (instead of `Checks if …`)
It is better to mention it returns true if the Command *potentially* mutates the data (header comments specify a contract, so try to be as precise as possible). E.g. Adding a duplicate person will not mutate data but the corresponding AddCommand object will still return true for this method.
@return … line can be omitted for this method as it says the same thing as the first sentence.